src/main/groovy/life/qbic/datamodel/datasets/datastructure/files/nfcore/UnspecifiedFile.groovy

```diff
@@ -0,0 +1,30 @@
+package life.qbic.datamodel.datasets.datastructure.files.nfcore
+
+import life.qbic.datamodel.datasets.datastructure.files.DataFile
+
+/**
+ * A specialisation of a DataFile, represents a non-specified/variable output file in the nf-core Pipeline
+ *
+ * @since 2.7.0
+ */
+class UnspecifiedFile extends DataFile {
+
+    protected UnspecifiedFile() {}
+
```

Comment:
Do we really want to allow all possible files? There could then also be files accepted that we do not want to have.